planning/behavior_path_planner/include/behavior_path_planner/scene_module/side_shift/side_shift_module.hpp

```diff
@@ -46,6 +50,27 @@ struct SideShiftParameters
   double drivable_area_height;
 };
 
+struct SideShiftRequestTimer
```

Comment:
The name(SideShiftRequestTimer) is supposed to send requests periodically, is that correct?

Reply:
The `SideShiftRequestTimer` doesn't send any request. It is just used track `lateral_offset` requests made by the scene module node.